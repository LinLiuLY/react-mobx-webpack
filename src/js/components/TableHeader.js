import React from 'react';

export default TableHeader => (
  <thead>
    <tr>
      <th>name</th>
    </tr>
  </thead>
);
