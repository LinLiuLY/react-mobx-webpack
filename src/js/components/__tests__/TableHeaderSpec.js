import React from 'react';
import TableHeader from '../TableHeader';
import renderer from 'react-test-renderer';

it('should render TableHeader', () => {
  const tableHeader = renderer.create(
    <TableHeader />
  ).toJSON();
  expect(tableHeader).toMatchSnapshot();
});
