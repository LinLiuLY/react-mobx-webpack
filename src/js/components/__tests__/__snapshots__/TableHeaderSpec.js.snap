// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TableHeader 1`] = `
<thead>
  <tr>
    <th>
      name
    </th>
  </tr>
</thead>
`;
